pycytominer/cyto_utils/DeepProfiler_processing.py

```diff
@@ -360,7 +360,32 @@ def __init__(
 
         self.deep_data = deep_data
 
-    def get_single_cells(self, output=False):
+    def get_single_cells_column(self, output=False, column_num=0):
+        # build filenames if they do not already exist
+        if not hasattr(self.deep_data, "filenames"):
+            self.deep_data.build_filenames()
+
+        total_features = np.ndarray(shape=(0, 1280), dtype=np.float32)
```

Comment:
This won't always be 1280, correct? It is probably best to set 1280 as a default function argument.